kview/src/main/resources/dev/ikm/komet/kview/mvvm/view/kview.css

```diff
@@ -2950,12 +2959,38 @@ Timeline Year line segment (will overlay change circles)
     -fx-font-weight: 400;
     -fx-background-color: white;
     -fx-effect: dropshadow(gaussian, rgba(0, 0, 0, 0.30), 1, 1, 1, 1);    
-    -fx-min-width: 280px;
-    -fx-pref-width: 280px;
-    -fx-max-width: 280px;
-    -fx-pref-height: 30;    
+    -fx-min-width: 285px;
+    -fx-pref-width: 285px;
+    -fx-max-width: 285px;
+    -fx-pref-height: 30;
+    -fx-padding: 5;
+    -fx-alignment: center;
 }
 
+.pattern-instance-hbox .pattern-instance {
+    -fx-min-width: 260px;
+    -fx-pref-width: 260px;
+    -fx-max-width: 260px;
+
+}
+.pattern-instance-hbox .pattern-instance-hover-icon {
+         visibility: hidden;
```

Comment:
small formatting issue: too much extra spaced before "visibility". It's also present below.